boards/acd52832/src/main.rs

```diff
@@ -22,6 +23,7 @@ use kernel::platform::{KernelResources, SyscallDriverLookup};
 use kernel::scheduler::round_robin::RoundRobinSched;
 #[allow(unused_imports)]
 use kernel::{create_capability, debug, debug_gpio, static_init};
+use nrf52832::deferred_call_tasks::DeferredCallTask;
```

Comment:
I think it would be nice to remove this import, as it would make the `dc_mgr` more clear that it is chip-specific.